tests/models/test_cookies.py

```diff
@@ -69,7 +69,7 @@ def test_multiple_set_cookie():
         ),
     ]
     request = httpx.Request("GET", "https://www.example.org")
-    response = httpx.Response(200, request=request, headers=headers)
+    response = httpx.Response(status_code=200, request=request, headers=headers)
```

Comment:
Do we want to keep these `status_code=200` in the sale of "explicit over implicit", or should we rely on the new "200 by default"?

Reply:
I think explicitly specifying `status_code=200` is better.